### ISSUE TITLE
ALC236 sleep/wake issue

### ISSUE THREAD

**Issue by coolstar** [vit9696/AppleALC#131]
When I have AppleALC installed on ALC236, the first sleep/wake works fine, however on subsequent sleep/wake attempts, the laptop instantly wakes up from sleep.

VoodooHDA does not have this problem.

**Comment by vit9696** [comment]
TL;DR: I do not think it is an AppleALC issue, since we do not temper with the sleep/wake code. And since it does not happen for most of the users including me.

In theory it could be caused by AppleHDA resources we use. I could suggest trying to install them manually and confirm the bug without AppleALC. Then you are best to go to the forums specialising in AppleHDA resource development, there you might be able to find something/someone that could you help you. Once you have the resources working for you, you are welcome to submit them here.

I also see no reason not to use VoodooHDA if it works for you. Sound quality is pretty much the same after all.
